default.ps1

```diff
@@ -1,21 +1,26 @@
 # Requires psake to run, see README.md for more details.
 
+Framework 4.6
+$InformationPreference = 'Continue'
+$env:PSModulePath += ";$PSScriptRoot\Scripts\Modules"
+
 if ($PSVersionTable.PSVersion.Major -lt 3)
 {
     throw "PowerShell 3 is required.`nhttp://www.microsoft.com/en-us/download/details.aspx?id=40855"
 }
 
-Framework 4.6
-$InformationPreference = 'Continue'
-$env:PSModulePath += ";$PSScriptRoot\scripts\Modules"
-
 . .\scripts\Saritasa.PsakeTasks.ps1
 
 . .\scripts\BuildTasks.ps1
+. .\scripts\DockerTasks.ps1
 . .\scripts\PublishTasks.ps1
 
-Import-Module Saritasa.Build
-Import-Module Saritasa.Test
+Properties `
+{
+    $Version = '0.1.0'
```

Comment:
It's used in CompileDocs function.

    (Get-Content "$docsRoot\conf.py").Replace('VX.VY', $Version) | Set-Content "$docsRoot\conf.py"